bool is_trash
float32 distance_cm
geometry_msgs/Point center

